lib/_stream_readable.js

```diff
@@ -56,10 +56,6 @@ function ReadableState(options, stream) {
   // Everything else in the universe uses 'utf8', though.
   this.defaultEncoding = options.defaultEncoding || 'utf8';
 
-  // when piping, we only care about 'readable' events that happen
-  // after read()ing all the bytes and not getting any pushback.
-  this.ranOut = false;
-
```

Comment:
We should probably leave this in for now. I'll add it to the list of estoc-able queries, though!